src/components/UriViewer.js

```diff
@@ -268,7 +268,7 @@ export const UriViewerLink = ({ uri, googleProject }) => {
         e.preventDefault()
         setModalOpen(true)
       }
-    }, [isGs(uri) ? _.last(uri.split('/')) : uri]),
+    }, [isGs(uri) ? _.last(uri.replace(/\/$/, '').split('/')) : uri]),
```

Comment:
```suggestion
    }, [isGs(uri) ? _.flow(_.split('/'), _.flatten, _.last)(uri) : uri]),
```

Reply:
What about this?

Reply:
As a bonus, this will retain a trailing slash.

Reply:
That seems to work - it just leaves the extra slash at the end (which I think is probs fine)

Reply:
Updated